src/main/resources/application.yml

```diff
@@ -57,6 +57,9 @@ micronaut:
             jwks-uri: ${oauth.openId.jwksUri:`https://sandbox.orcid.org/oauth/jwks`}
             user-info:
               url: ${oauth.openId.userInfoUrl:`https://sandbox.orcid.org/oauth/userinfo`}
+      state:
+        cookie:
+          cookie-max-age: 10m
```

Comment:
Why was this added?

Reply:
That increases the nonce token timeout to 10m instead of 5m. It's not necessary but adds a little more flexibility for the user.